native/acter/src/api/attachments.rs

```diff
@@ -476,9 +480,11 @@ impl AttachmentsManager {
 
         RUNTIME
             .spawn(async move {
-                trace!("before redacting attachment");
+                let permitted = room.can_user_redact_own(&my_id).await?;
+                if !permitted {
+                    bail!("No permissions to redact your message in this room");
+                }
```

Comment:
if we are already adding this here, we need to make the proper check:
if the event is by _this_ user we need to check for `can_user_redact_own` and if it is not we need to check for `can_user_redact_other`.

This basically applies to all _redacts_ ...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1671/commits/3ec8b833f13bc2974c6a896e7223b64068b081b3